android/src/main/java/com/oney/WebRTCModule/GetUserMediaImpl.java

```diff
@@ -390,6 +393,39 @@ private VideoTrack createVideoTrack(AbstractVideoCaptureController videoCaptureC
         return track;
     }
 
+    
+    void setVideoEffect(String trackId, String name) {
+        TrackPrivate track = tracks.get(trackId);
+
+        if (track != null && track.videoCaptureController instanceof CameraCaptureController) {
+            VideoSource videoSource = (VideoSource) track.mediaSource;
+            SurfaceTextureHelper surfaceTextureHelper = (SurfaceTextureHelper) track.surfaceTextureHelper;
+            /// here set videoSource processer VideoProcessor
+
+            if (surfaceTextureHelper == null) {
+                Log.d(TAG, "Error creating SurfaceTextureHelper");
```

Comment:
unnecessary check. We fail to create the track if we can't create the surface texture helper, so here you can assume it will always exist.